src/main/java/uk/gov/hmcts/reform/logging/exception/AbstractLoggingException.java

```diff
@@ -1,6 +1,5 @@
 package uk.gov.hmcts.reform.logging.exception;
 
-import ch.qos.logback.classic.spi.ILoggingEvent;
```

Comment:
do you want to bump the version here, or a separate change?

Reply:
Would be nice to bump here and have it all done.

Reply:
you meant overall pr, not this single line change, right?